### ISSUE TITLE
setting a namespace option to nil

### ISSUE THREAD

**Issue by kellyredding** [redding/ns-options#18]
I have a namespace option, for example:

```
options :config do
  option :root, Pathname, :default => '.'
end
```

the value is its default, as expected:

```
something.config.root  #=> #<Pathname:.>
```

I set it equal to nil, but the value is still the orig default:

```
something.config.root  #=> #<Pathname:.>
```

I would expect the value returned to be nil in this case.  I'm guessing this is a bug?

**Comment by kellyredding** [comment]
@jcredding - here is the writeup of that bug I found.

**Comment by kellyredding** [comment]
This should be closed when issue #21 is closed.
